src/Nox.Generator/Application/ServiceCollectionExtensionsGenerator.cs

```diff
@@ -6,7 +6,11 @@ namespace Nox.Generator.Application;
 
 internal class ServiceCollectionExtensionsGenerator : INoxCodeGenerator
 {
-    public NoxGeneratorKind GeneratorKind => NoxGeneratorKind.None;
+    public NoxGeneratorKind GeneratorKind => 
```

Comment:
Should be refactored in the future for proper generation of ServiceCollectionExtensionsGenerator.cs

Reply:
IMO you should set has Application, so the Application layer is responsible to services registration, and so on.... actually this one is already in the proper namespace